### PR TITLE
Add drive invitation on readme

### PR DESCRIPTION
**Pull request by amrita-shrestha**
This PR adds drive invitation-related part on the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,45 @@ $drives[0]->uploadFile("/documents/myfile.txt", "Hello World!");
 $resources = $drives[0]->getResources("/documents");
 ```
 
+### Drive Permission
+Users/Groups can be invited to drives by specifying permissions. Drive class has method to update permission roles and expiration dates, remove users and groups, etc. related to drive invitations.
```
Comment (individual-it) [inline README.md:92]:
```suggestion
Users/Groups can be invited to drives by specifying permissions. The `Drive` class has methods to invite Users/Groups, update permission roles and expiration dates, remove users and groups, etc.
```

File: README.md [hunk 2]
```diff
@@ -88,6 +88,45 @@ $drives[0]->uploadFile("/documents/myfile.txt", "Hello World!");
...
+
+Drive invitations are allowed only on the **project drive**. For drive invitation, **root endpoint** has been used.
```
Comment (individual-it) [inline README.md:94]:
```suggestion
Drive invitations are only possible on **project drives**.
```

I don't think the user of the SDK needs to know what endpoint is used

File: README.md [hunk 3]
```diff
@@ -88,6 +88,45 @@ $drives[0]->uploadFile("/documents/myfile.txt", "Hello World!");
...
+$users = $ocis->getUsers("einstein")[0];
+
+// get all drives of type project
+$drives = $ocis->getMyDrives(
+    DriveOrder::NAME,
+    OrderDirection::ASC,
+    DriveType::PROJECT
+);
+
+// get project drive name game
```
Comment (individual-it) [inline README.md:107]:
```suggestion
// get the drive named 'game'
```

File: README.md [hunk 4]
```diff
@@ -88,6 +88,45 @@ $drives[0]->uploadFile("/documents/myfile.txt", "Hello World!");
...
+    if ($drive->getName) === 'game' {
+        $gameDrive = $drive;
+        break;
+    }
+}
+
+// get all roles that are possible for that drive
+$driveRoles = $gameDrive->getRoles();
+
+// get role that allowed to view, download, upload, edit, add, delete and manage members
```
Comment (individual-it) [inline README.md:118]:
```suggestion
// get the role that is allowed to view, download, upload, edit, add, delete and manage members
```

**Review by individual-it — COMMENTED** [review]
Just some language comments, see what you think makes sense

**Review by phil-davis — APPROVED** [review]

**Pull request merged**